scripts/malloc-init.pl

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/perl
```

Comment:
Well, I am confident that the script is not used in any of our scripts, but I cannot say about our documentation. There are too many articles to check by hand. Do you think greping our documentation repository is sufficient?